main.go

```diff
@@ -105,6 +110,9 @@ func runSQL(quitContext context.Context, db database, sql string, key string, pr
 
 	mysql := "mysql"
 	options := fmt.Sprintf(" -Nsr %v%v%v%v -e ", userOption, passOption, hostOption, db.DbName)
+	if *pretty {
+		options = fmt.Sprintf(" -vt %v%v%v%v -e ", userOption, passOption, hostOption, db.DbName)
```

Comment:
If I understand this correctly, the only difference is `-Nsr` -> `-vt`. Could you please make this yet another `%v` rather than repeating the whole `options` line? i.e. creating `flagsOption` beforehand.